gtfs_digest/_gtfs_digest_specific_datasets.py

```diff
@@ -45,7 +45,13 @@ def get_day_type(date):
     """
     Function to return the day type (e.g., Monday, Tuesday, etc.) from a datetime object.
     """
-    days_of_week = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
+    days_of_week = ["Monday", 
+                    "Tuesday", 
+                    "Wednesday", 
+                    "Thursday", 
+                    "Friday", 
+                    "Saturday", 
+                    "Sunday"]
     return days_of_week[date.weekday()]
 
 def weekday_or_weekend(row):
```

Comment:
Already a function: https://github.com/cal-itp/data-analyses/blob/main/rt_segment_speeds/segment_speed_utils/gtfs_schedule_wrangling.py#L112

Reply:
Thanks for linking the function. I think I can't use it since I have to differentiate Saturday and Sunday for the charts, instead of grouping together under "weekend."